fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+#define SIZE 100000
+
+struct FenwickTree {
+    vector<int> v;
+    int len;
+
+    FenwickTree(int len = SIZE) : len(len) {
+        v.assign(len + 1, 0);
+    }
+
+    void update(int, int);
+    int query(int, int);
+    int query(int);
+};
+
+void FenwickTree::update(int i, int val) {
```

Comment:
Constructors callable with one argument should be marked explicit.  [runtime/explicit] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*